aspnetcore/client-side/libman/libman-cli.md

```diff
@@ -0,0 +1,498 @@
+
+```console
+libman
+```
+
+To view the installed CLI version:
+
+```console
+libman --version
+```
+
+To view the available CLI commands:
+
+```console
+libman --help
+```
+
+The following sections outline the available CLI commands.
+
+## Initialize LibMan in the project
+
+The `libman init` command creates a *libman.json* file if one doesn't exist. The file is created with the default item template content.
+
+### Synopsis
+
+```console
+libman init [-d|--default-destination] [-p|--default-provider] [--verbosity]
+libman init [-h|--help]
+```
+
+### Options
+
+The following options can be appended to the `libman init` command:
+
+* `-d|--default-destination <PATH>`
+
+  A path relative to the current folder. Library files are installed in this location if no `destination` property is defined for a library in *libman.json*. The `<PATH>` value is written to the `defaultDestination` property of *libman.json*.
+
+* `-p|--default-provider <PROVIDER>`
+
+  The provider to use if no provider is defined for a given library. The `<PROVIDER>` value is written to the `defaultProvider` property of *libman.json*. Replace `<PROVIDER>` with one of the following values:
+
+  [!INCLUDE [LibMan provider names](../../includes/libman-cli/provider-names.md)]
+
+[!INCLUDE [standard-cli-options](../../includes/libman-cli/standard-cli-options.md)]
+
+### Examples
+
+To create a *libman.json* file in an ASP.NET Core project:
+
+* Navigate to the project root.
+* Run the following command:
+
+  ```console
+  libman init
+  ```
+
+* Type the name of the default provider to be used, or press `Enter` to use the default CDNJS provider. Valid values include:
+
+  [!INCLUDE [LibMan provider names](../../includes/libman-cli/provider-names.md)]
+
+A *libman.json* file is added to the project root with the following content:
+
+```json
+{
+  "version": "1.0",
+  "defaultProvider": "cdnjs",
+  "libraries": []
+}
+```
+
+## Add library files
+
+The `libman install` command downloads and installs library files into the project. A *libman.json* file is added if one doesn't exist. The *libman.json* file is modified to store configuration details for the library files.
+
+### Synopsis
+
+```console
+libman install <LIBRARY> [-d|--destination] [--files] [-p|--provider] [--verbosity]
+libman install [-h|--help]
+```
+
+### Arguments
+
+`LIBRARY`
+
+The name of the library to install. This name may include version number notation (for example, `@1.2.0`).
+
+### Options
+
+The following options can be appended to the `libman install` command:
+
+* `-d|--destination <PATH>`
+
+  The location to install the library. If not specified, the default location is used. If no `defaultDestination` property is specified in *libman.json*, this option is required.
+
+* `--files <FILE>`
+
+  The name of the file to be installed from the specified library. If not specified, all files from the library are installed. Provide one `--files` option per file to be installed.
```

Comment:
> Specifies the name of the file to install from the specified library.

Can you tell that my mentors only permitted state verbs on :skull: **_pain of death_!** :skull: :smile: ... We could be rebels, but I'm still afraid of Henderson. Trust me on this ... he **will** rise from the grave and come after us. 🏃😨